recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   md5: bb53fd06f7ebb74589e9dd684b1b66a6
```

Comment:
Changing from `1.5.6` to `1.5.11` the hash should be changed accordignly
```diff
- md5: bb53fd06f7ebb74589e9dd684b1b66a6
+ md5: a79cf9d1200f37e0466062c3ce396d6b
```

Reply:
Oops